README.md

```diff
@@ -120,6 +119,21 @@ If the `cds-maven-plugin` is used correctly and executed the following lines sho
 
 After that the models can be used.
 
+### Extend db entity in srv module
+
+To avoid making the `db` a Maven artifact with a `db/pom.xml`, you can alternatively extend the db entity from the CDS model in the srv module.
```

Comment:
```suggestion
To avoid making the `db` a Maven artifact with a `db/pom.xml`, you can alternatively extend the db entity from the CDS model in the srv module.
```
```suggestion
To avoid making the `db` a Maven artifact with a `db/pom.xml`, you can alternatively extend the db entity from the CDS model in the srv module.
```
```suggestion
To avoid making the `db` a Maven artifact with a `db/pom.xml`, you can alternatively extend the `db` entity from the CDS model in the `srv` module.
```